# All units are in SI
# ===================
# Time:     second, hz
# Angle:    radian
# Distance: meter

/**:
  ros__parameters:
    world_frame: "map"
<<<<<<< Updated upstream
    gps_frame: "gps_frame"
=======
    rover_frame: "right_gps_frame"

    # Exception, these are in degrees
    ref_lat: 51.4668369 #38.4227303 
    ref_lon: -112.7051763 #-110.7847932
    ref_alt: 0.0
>>>>>>> Stashed changes

basestation_gps_driver:
  ros__parameters:
    port: /dev/ublox_gps
    baud: 38400

rover_gps_driver:
  ros__parameters:
    port_unicore: /dev/unicore_gps
    baud_unicore: 115200

    port_ublox: /dev/ublox_gps
    baud_ublox: 38400

heading_filter:
  ros__parameters:
    imu_watchdog_timeout: 1.0
    mag_heading_noise: 1.0
    rtk_heading_noise: 0.01
    process_noise: 0.000001

pose_filter:
  ros__parameters:
    rover_heading_change_threshold: 0.05
    minimum_linear_speed: 0.4
